app/models/session.rb

```diff
@@ -25,7 +25,8 @@
 #  fk_rails_...  (organisation_id => organisations.id)
 #
 class Session < ApplicationRecord
-  audited
+  audited associated_with: :location
```

Comment:
I wonder if associating these with the organisation would be more useful, what do you think?

Reply:
Ya, could be. The reason I chose `location` in the end was mainly because I thought how locations can have multiple sessions, so trying to understand what's happened to a location and it's sessions could be useful from an ops pov.  There's also the association link that goes `organisation` <- `team` <- `location` <- `session`, which while long at least is there. We could conceivably write something that pulls up all the associated audits pretty easily since there won't be any loops.

Reply:
Yeah that's fair enough. I think one tricky situation could be if a school changes management to a different SAIS organisation, but whether that would ever happen I'm not sure.

Reply:
Oof. That's an interesting one. It feels like that might be a bit in the future, happy to revisit/revise this as we go.